cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-ui.adoc

```diff
@@ -0,0 +1,316 @@
+
+== Creating account wide roles
+
+[NOTE]
+====
+If you already have account roles from an earlier deployment, skip this step. The UI will detect your existing roles after you select an associated AWS account.
+====
+
+If this is the _first time_ you are deploying ROSA in this account and you have _not_ yet created the account roles, create the account-wide roles and policies, including the Operator policies.
+
+* In your terminal, run the following command to create the account-wide roles:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode auto --yes
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Creating roles using 'arn:aws:iam::000000000000:user/rosa-user'
+I: Created role 'ManagedOpenShift-ControlPlane-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-ControlPlane-Role'
+I: Created role 'ManagedOpenShift-Worker-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-Worker-Role'
+I: Created role 'ManagedOpenShift-Support-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-Support-Role'
+I: Created role 'ManagedOpenShift-Installer-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-Installer-Role'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-machine-api-aws-cloud-credentials'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-cloud-credential-operator-cloud-crede'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-image-registry-installer-cloud-creden'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-ingress-operator-cloud-credentials'
+I: Created policy with ARN 'arn:aws:iam::000000000000:policy/ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credent'
+I: To create a cluster with these roles, run the following command:
+rosa create cluster --sts
+----
+
+== Associating your AWS account with your Red Hat account
+This step tells the OpenShift Cluster Manager what AWS account you want to use when deploying ROSA.
+
+[NOTE]
+====
+If you have already associated your AWS accounts, skip this step.
+====
+
+. Open the OpenShift Cluster Manager by visiting the Red Hat link:https://console.redhat.com/openshift[console] and logging in to your Red Hat account.
+
+. Click *Create Cluster*.
+
+. Scroll down to the {product-title} (ROSA) row and click *Create Cluster*.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-create.png[]
+
+. A dropdown menu appears. Click *With web interface*.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-web-interface.png[]
+
+. Under "Select an AWS control plane type", choose *Classic*. Then click *Next*.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-classic.png[]
+
+. Click the dropbox under *Associated AWS infrastructure account*. There might not be any associated accounts if you have not yet associated any AWS accounts. 
```

Comment:
Maybe change wording to "If you have not yet associated any AWS accounts, this dropdown will be blank."